File: JavaScript/2025-12-08.js
function convertToKgs(lbs) {
    const kgs = (lbs * 0.453592).toFixed(2);

    if (lbs === 1) {
        return `${lbs} pound equals ${kgs} kilograms.`
    }

    else if (parseFloat(kgs) === 1) {
        return `${lbs} pounds equals ${kgs} kilogram.`
    }

    else {
        return `${lbs} pounds equals ${kgs} kilograms.`
    }

}

File: Python/2025-12-08.py
def convert_to_kgs(lbs):
    kgs = format(lbs * 0.453592, '.2f')

    if lbs == 1:
        return f"{lbs} pound equals {kgs} kilograms."

    elif float(kgs) == 1:
        return f"{lbs} pounds equals {kgs} kilogram."

    else:
        return f"{lbs} pounds equals {kgs} kilograms."